testing/util-tls.go

```diff
@@ -0,0 +1,129 @@
+package testing
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func CreateLocalTLSData(identityName string) error {
```

Comment:
I agree; however, we are using the same function in the example if the user of the example would like to generate quick TLS data and there is no testing abstraction. It's possible that some of the `testing utils` just need to move out to just `util`.

Reply:
I'd definitely try to avoid a `util` package - I find that they become a fairly generic dumping ground for unrelated code and this quickly grows in scope. Perhaps a `tls` package or similar would make more sense ?